package.json

```diff
@@ -21,7 +26,7 @@
     "@types/react-dom": "^18.0.11",
     "@typescript-eslint/eslint-plugin": "^5.57.1",
     "@typescript-eslint/parser": "^5.57.1",
-    "@vitejs/plugin-react-swc": "^3.0.0",
+    "@vitejs/plugin-react": "^4.0.0",
```

Comment:
The swc plugin doesn't seem to support web-components just yet. They don't seem to be bundled in correctly.